cmd/dcrctl/version.go

```diff
@@ -27,9 +27,9 @@ const (
 )
 
 // appBuild is defined as a variable so it can be overridden during the build
-// process with '-ldflags "-X main.appBuild foo' if needed.  It MUST only
+// process with '-ldflags "-X main.appBuild=foo' if needed.  It MUST only
 // contain characters from semanticAlphabet per the semantic versioning spec.
-var appBuild string
+var appBuild = "dev"
```

Comment:
hm interesting, I could have sworn it worked.  I should probably check for my own usage of this trick to makes sure i'm doing it correctly